### ISSUE TITLE
Leftover Apache Commons StringUtils requirement?

### ISSUE THREAD

**Issue by poikilotherm** [kordamp/pomchecker#4]
Hi there,

when trying to use this neat project with https://github.com/ocpsoft/rewrite/issues/351, I came across this problem when using the Maven plugin:

```
[ERROR] Failed to execute goal org.kordamp.maven:pomchecker-maven-plugin:1.3.0:check-maven-central (default-cli) on project rewrite-parent: Execution default-cli of goal org.kordamp.maven:pomchecker-maven-plugin:1.3.0:check-maven-central failed: A required class was missing while executing org.kordamp.maven:pomchecker-maven-plugin:1.3.0:check-maven-central: org/apache/commons/lang3/StringUtils
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>org.kordamp.maven:pomchecker-maven-plugin:1.3.0
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
[ERROR] urls[0] = file:/home/obertuch/.m2/repository/org/kordamp/maven/pomchecker-maven-plugin/1.3.0/pomchecker-maven-plugin-1.3.0.jar
[ERROR] urls[1] = file:/home/obertuch/.m2/repository/org/apache/maven/plugin-tools/maven-plugin-annotations/3.6.4/maven-plugin-annotations-3.6.4.jar
[ERROR] urls[2] = file:/home/obertuch/.m2/repository/org/kordamp/maven/pomchecker-core/1.3.0/pomchecker-core-1.3.0.jar
[ERROR] urls[3] = file:/home/obertuch/.m2/repository/org/codehaus/plexus/plexus-utils/1.1/plexus-utils-1.1.jar
[ERROR] Number of foreign imports: 1
[ERROR] import: Entry[import  from realm ClassRealm[project>org.ocpsoft.rewrite:rewrite-parent:6.0.0-SNAPSHOT, parent: ClassRealm[maven.api, parent: null]]]
[ERROR] 
[ERROR] -----------------------------------------------------
```

Looking at the code, there is only one class using this import at https://github.com/kordamp/pomchecker/blob/f593b51afe95459083371e47375d1256c0ae158b/pomchecker-core/src/main/java/org/kordamp/maven/checker/MavenCentralChecker.java#L32-L33

Is this project missing a dependency? Many projects probably have this dependency on their classpath... Is it necessary to use this external dependency? I see no guarantees for supported Java versions, so maybe this can be replaced by JDK builtins?

Thank you and keep up the good work!

**Comment by poikilotherm** [comment]
Now that was fast! Thanks a ton @aalmiray !

**Comment by aalmiray** [comment]
Released in `1.4.0`
